.github/workflows/tests.yaml

```diff
@@ -33,6 +33,16 @@ jobs:
           cache: 'pip'
       - name: Configure environment
         run: |
+          echo "::group::Remove unnecessary packages"
```

Comment:
You should be able to drop this from the PR now.  It is supposed to be for ` integration-tests-linux:` instead of `linters`.  Perhaps that's why it didn't initially work